libs/langgraph/langgraph/func/__init__.py

```diff
@@ -178,6 +186,11 @@ def entrypoint(
         config_schema: Specifies the schema for the configuration object that will be
             passed to the workflow.
 
+    Attributes:
+        final: A special attribute that can be used to return a value and save a value
+            to the checkpointer. This is useful when you want to return a value from
+            the function and save a value to the check
```

Comment:
I think this should say something like "save a value to the checkpointer distinct from the one returned"